src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
```

Comment:
For pac4j, the clients are always more important because, by default, we expect the user to be authenticated (so it's implicitly "isAuthenticated"). That's why I don't think we should use the `value` property.